workers/data_refinery_workers/downloaders/test_array_express.py

```diff
@@ -148,7 +148,7 @@ def test_verification_failure(self,
         self.assertIsNotNone(downloader_job.start_time)
         self.assertIsNotNone(downloader_job.end_time)
         self.assertEqual(downloader_job.failure_reason,
-                         "A Batch doesn't have the same download URL as the other batches")
+                         "A Batch doesn't have the same download URL as the other batches.")
```

Comment:
Would it make sense to output which Batch doesn't have the same download URL in the failure_reason of the downloader_job?

Reply:
I thought about that, however the issue is that it's not clear which is the culprit Batch. They could all have different URLs which would then make calling one of them out as being wrong a red herring. I think the correct debugging steps when seeing that `failure_reason` are to look at all the `download_urls` for the Batches in the DownloaderJob and I think the ambiguity would perhaps lead the person debugging it to look at all of them.